src/main/java/org/minions/devfund/richard/battleship/BattleCruiser.java

```diff
@@ -0,0 +1,36 @@
+package org.minions.devfund.richard.battleship;
+
+/**
+ * Class.
+ */
+public class BattleCruiser extends Ship {
+
+    private static final int SHIP_NUM = 1;
+    private static final String SHIP_TYPE = "battlecruiser";
+    private static final int SHIP_LENGTH = 7;
+
+    /**
+     * Constructor.
+     */
+    BattleCruiser() {
+        super.setLength(SHIP_LENGTH);
+        super.setHit(new boolean[SHIP_LENGTH]);
+        super.setNumber(SHIP_NUM);
+    }
+
+    /**
+     * {@inheritDoc}.
+     */
+    @Override
+    public String getShipType() {
+        return SHIP_TYPE;
+    }
+
+    /**
+     * {@inheritDoc}.
+     */
+    @Override
+    Ship getShip() {
+        return new BattleCruiser();
```

Comment:
@RichardSeverich 
This method should be removed from all your child ship classes.